charts/ldap/templates/persistentVolumeClaim.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "ldap.fullname" . }}-backup
+spec:
+  storageClassName: ""
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Gi
+  selector:
+    matchLabels:
+      role: ldap-backup
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "ldap.fullname" . }}-data
```

Comment:
I can't find the PV for this in the PR? is it defined somewhere else?

Reply:
This is the default behavior, Kubernetes provisions a new azure disk storage and then attach it to the pod, which means that the lifecycle is tied to kubernetes cluster.

![image](https://user-images.githubusercontent.com/2360224/71979053-11a9c300-321d-11ea-88c6-dbea968a86d2.png)

More information on this application design [here](https://github.com/jenkins-infra/iep/tree/master/iep-008#data)

Reply:
Ah, I see. thanks for the link, didn't realise the backup/restore was so fast